nemo/collections/nlp/data/token_classification/punctuation_capitalization_dataset.py

```diff
@@ -382,8 +382,8 @@ def create_label_ids(unique_labels, pad_label=pad_label):
         self.all_input_ids = features[0]
         self.all_segment_ids = features[1]
         self.all_input_mask = features[2]
-        self.all_loss_mask = features[3]
-        self.all_subtokens_mask = features[4]
+        self.all_subtokens_mask = features[3]
+        self.all_loss_mask = features[4]
```

Comment:
wow, how the training could even work with masks messed up...?

Reply:
@tkornuta-nvidia depending on the parameters, the masks could be the same. This fix won't require retraining the models.